ledger/complete/ledger.go

```diff
@@ -265,7 +267,8 @@ func (l *Ledger) Checkpointer() (*wal.Checkpointer, error) {
 func (l *Ledger) ExportCheckpointAt(
 	state ledger.State,
 	migrations []ledger.Migration,
-	reporters []ledger.Reporter,
+	reporters map[string]ledger.Reporter,
```

Comment:
I'm not sure if I follow the change here, if we are interested in access to name of reporter we could add Name as a method to reporter interface